e2e/transactions/SendNft.yaml

```diff
@@ -0,0 +1,76 @@
+appId: ${APP_ID}
+tags:
+  - wallet
+  - parallel
+---
+- retry:
+    maxRetries: 3
+    commands:
+      - launchApp:
+          clearState: true
+          clearKeychain: true
+          arguments:
+            isE2ETest: true
+      - runFlow: ../utils/Prepare.yaml
+      - runFlow: ../utils/ImportWalletWithKey.yaml
+
+      # Send ETH to test wallet
+      - tapOn:
+          id: fund-test-wallet-button
+
+      # Connect to Anvil (test network)
+      - tapOn:
+          id: dev-button-anvil
+
+      # Open send sheet
+      - retry:
+          maxRetries: 3
+          commands:
+            - assertVisible:
+                id: 'balance-coin-row-Ethereum'
+
+      - tapOn: '.*Rainbow Pooly.*'
+
+      - assertVisible:
+          id: 'wrapped-nft-Rainbow Pooly 104'
+
+      - tapOn:
+          id: 'wrapped-nft-Rainbow Pooly 104'
```

Comment:
I don't think we need both assertVisible and tapOn, just the tapOn should be enough since maestro automatically waits for it to be visible before tapping.